pylzy/tests/api/v1/test_calls.py

```diff
@@ -0,0 +1,242 @@
+from typing import Optional, List, Tuple
+from unittest import TestCase, skip
+
+from pydantic import ValidationError
+
+from lzy.api.v1 import Lzy, op
+from lzy.api.v1.signatures import FuncSignature
+from lzy.api.v1.utils.proxy_adapter import materialized
+from tests.api.v1.mocks import RuntimeMock, StorageRegistryMock
+
+
+@op(description="aaa")
+def returns_int() -> int:
+    return 1
+
+
+# noinspection PyUnusedLocal
+@op
+def one_arg(arg: int) -> str:
+    pass
+
+
+# noinspection PyUnusedLocal
+@op
+def one_default(arg: int, b: Optional[str] = None) -> str:
```

Comment:
I would test default arg without optional, and for Optional type I would create another test.

Reply:
agree